docs/docs/guides/wagmi_usage/wagmi.md

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 1
+sidebar_label: 'Wagmi Web3js Adaptor'
+title: 'Wagmi Web3js Adaptor'
+---
+
+If you're using [Wagmi](https://wagmi.sh/react/getting-started#use-wagmi) and want to add web3.js, use this provider in your project:
```

Comment:
lgtm, just add some more description for this part that these are adapter snippets